### DISCUSSION TITLE
Podman Compose

### DISCUSSION THREAD

**Discussion by trymeouteh** [podman-desktop/podman-desktop#5397] [Ideas]
Create "podman-compose" which will be the same as docker compose but within podman desktop as a GUI.

- Create a compose in the podman desktop GUI
- Import docker compose files in podman desktop
- Export compose into docker compose files from podman desktop
- create and delete a compose in podman desktop GUI (compose up, compose down)

Podman compose feature requeat for podman cli
https://github.com/containers/podman/discussions/21116

**Comment by afbjorklund** [comment]
Podman Desktop currently uses `docker-compose`, as installed by the "Compose" extension

https://podman-desktop.io/docs/compose/setting-up-compose
